test/periphery/PixCashierUpgradeable.test.ts

```diff
@@ -95,21 +103,27 @@ describe("Contract 'PixCashierUpgradeable'", async () => {
     it("Is reverted if the contract is paused", async () => {
       await proveTx(pixCashier.setPauser(deployer.address));
       await proveTx(pixCashier.pause());
-      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount))
+      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount, TRANSACTION_ID))
         .to.be.revertedWith(REVERT_MESSAGE_IF_CONTRACT_IS_PAUSED);
     });
 
     it("Is reverted if the user has not enough tokens", async () => {
       await proveTx(brlcMock.mint(cashierClient.address, tokenAmount - 1));
-      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount))
+      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount, TRANSACTION_ID))
         .to.be.revertedWith(REVERT_MESSAGE_IF_TOKEN_TRANSFER_AMOUNT_EXCEEDS_BALANCE);
     });
 
+    it("Is reverted if the transaction id is not provided", async () => {
+      await proveTx(brlcMock.mint(cashierClient.address, tokenAmount));
+      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount, ethers.constants.HashZero))
```

Comment:
Codacy has a fix for the issue: [Replace `pixCashier.connect(cashierClient).cashOut(tokenAmount,·ethers.constants.HashZero))` with `⏎········pixCashier⏎··········.connect(cashierClient)⏎··········.cashOut(tokenAmount,·ethers.constants.HashZero)`](https://app.codacy.com/gh/cloudwalk/brlc-token/pullRequest?prid=9795032)
```suggestion
      await expect(
```